src/TumblThree/TumblThree.Applications/Crawler/TumblrBlogCrawler.cs

```diff
@@ -440,6 +440,7 @@ private async Task AddUrlsToDownloadListAsync(TumblrApiJson document)
                 if (!PostWithinTimeSpan(post)) { continue; }
                 if (!CheckIfContainsTaggedPost(post)) { continue; }
                 if (!CheckIfDownloadRebloggedPosts(post)) { continue; }
+                string postTimestamp = post.Date;
```

Comment:
Why is this line needed?

Reply:
It is not. I forgot to remove it. It has been deleted now.